tests/epyccel/test_epyccel_variable_annotations.py

```diff
@@ -166,3 +174,107 @@ def homogeneous_tuple_annotation():
 
     assert epyc_homogeneous_tuple_annotation() == homogeneous_tuple_annotation()
     assert isinstance(epyc_homogeneous_tuple_annotation(), type(homogeneous_tuple_annotation()))
+
+@pytest.mark.parametrize( 'language', [
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="Variable declaration not implemented in fortran"),
```

Comment:
Please reference the issue that will fix this (this makes it easier to find all xfails that should be removed when you fix the issue as you just need to search for the number)

Reply:
updated cd22d9552fa7c174faa97aaba068fe8ff2acd247